### PR TITLE
Uncurry dcan in iset.mm

### PR DESCRIPTION
**Pull request by david-a-wheeler**
This adds a "dcan2" in curried form. This makes
transition easier.

### CODE REVIEW

**Comment by david-a-wheeler** [comment]
This is a trivial first step, but it's a step!

**Review by jkingdon — APPROVED** [review]
I tend to leave off the "Revised by" for changes this small but that's a non-blocking comment. You might know as much as I do what people would expect.

File: iset.mm [hunk 1]
```diff
@@ -7501,12 +7501,18 @@ converse also holds (see ~ pm5.6dc ).  (Contributed by Jim Kingdon,
   $}
 
   $( A conjunction of two decidable propositions is decidable.  (Contributed by
-     Jim Kingdon, 12-Apr-2018.) $)
-  dcan $p |- ( DECID ph -> ( DECID ps -> DECID ( ph /\ ps ) ) ) $=
+     Jim Kingdon, 12-Apr-2018.)  (Revised by David A. Wheeler, 12-Nov-2023.) $)
```
Comment (icecream17) [inline iset.mm:7504]:
```suggestion
     Jim Kingdon, 12-Apr-2018.)  (Revised by David A. Wheeler, 12-Nov-2024.) $)
```

I would assume the revised by would be on just dcan, but I'm newer so idk

**Review by icecream17 — APPROVED** [review]

**Review by avekens — APPROVED** [review]

**Comment by david-a-wheeler** [comment]
I wasn't sure if I should include "Revised by..." myself, since this is so trivial. Let's just remove that note. @jkingdon did the real lifting here, for this change I'm a drive-by trying to be mildly useful :-).

**Pull request merged**